.adops/package-update.yml

```diff
@@ -0,0 +1,89 @@
+trigger: none
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+  demands: project_navy
+
+resources:
+  pipelines:
+    - pipeline: packageRelease
+      source: 'Package release'
+      trigger:
+        branches:
+          include:
+            - master
+        stages:
+          - Prerelease
```

Comment:
may I know what's the purpose of this bit?

Reply:
@EmperorRXF here we are defining the `Prerelease` stage in the `Package release` pipeline as a source when that pipeline has been triggered off of the master branch.

Reply:
got it! had a hunch but thanks for clarifying. was wondering where the connection exists between the two pipelines to trigger this one. also one last thing, will the trigger happen only if that stage is successful or it would not care about that but just trigger this pipeline after "any" result is received on that stage?

Reply:
The stage(i.e: `Prerelease`) has to be successful in the source pipeline for the trigger event to kick-off :)